site/layouts/shortcodes/orange-footer.html

```diff
@@ -168,7 +174,11 @@ <h3 class="accordion-header footer-heading" id="headingFour{{ $id }}">
       <li class="fw-bold">© Orange {{ now.Format "2006" }}</li>
       <li><a class="nav-link" href="#">Terms and conditions</a></li>
       <li><a class="nav-link" href="#">Privacy</a></li>
-      <li><a class="nav-link" href="#">Accessibility statement</a></li>
+      {{- if or $in_examples $service }}
+        <li><a class="nav-link" href="#">Accessibility statement</a></li>
+      {{- else }}
+        <li><a class="nav-link active" href="#" aria-current="page">Accessibility statement</a></li>
```

Comment:
```suggestion
      <li><a class="nav-link active" href="#" aria-current="page">Accessibility statement</a></li>
```
Otherwise, feels weird on the highlight in the doc.